api/tests/test_asset_api.py

```diff
@@ -104,15 +104,19 @@ def test_authenticated_user_get_single_asset(self, mock_verify_id_token):
         self.assertEqual(response.status_code, 200)
 
     @patch('api.authentication.auth.verify_id_token')
-    def test_assets_api_endpoint_cant_allow_post(self, mock_verify_id_token):
+    def test_can_post_asset(self, mock_verify_id_token):
         mock_verify_id_token.return_value = {'email': self.user.email}
+        data = {
+            "asset_code": "IC002",
+            "serial_number": "SN002",
+            "model_number": self.assetmodel.id,
+        }
         response = client.post(
             self.asset_urls,
-            HTTP_AUTHORIZATION="Token {}".format(self.token_user)
-        )
-        self.assertEqual(response.data, {
-            'detail': 'Method "POST" not allowed.'
-        })
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
```

Comment:
Also check for an exact match. That is, make sure the response value corresponds to data.

Reply:
cool.on it

Reply:
@nelmandela Added this already.

Reply:
cool